revert: disable portuguese

naomi-lgbt opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->
First certification is not 100% yet. Shipped too soon. :(

Removing the cert from is-audited and the language from `availableLangs` will prevent the language from appearing as an option in the nav menu. Should be okay to leave the rest of the config present, as well as the challenge files, to streamline the process when we are ready to ship.

Commit:
revert: disable portuguese

First certification is not 100% yet. Shipped too soon. :(

raisedadead reviewed (commented):
On second thoughts, I think we can disable the language from the menu only (`availableLangs.client`) and still soft launch. 

And while we are here, I think the static list of languages in the is-audited files should be moved to the config instead.

nhcarrigan commented:
> On second thoughts, I think we can disable the language from the menu only (availableLangs.client) and still soft launch.

Sure, though we'd need to build the client in English and the curriculum in Portuguese for that deployment until we enable the client language again.

> And while we are here, I think the static list of languages in the is-audited files should be moved to the config instead.

I can look in to moving this. :)

Commit:
feat: enable portuguese curriculum

Curriculum can build in portuguese, leaving client disabled to
prevent language from appearing in nav bar.

Commit:
feat: move audited cert list

Moves the audited cert list to the config, instead of the utils.
